### PR TITLE
Changes in make and Dockerfile for Clone service, changes in tests

### PR DESCRIPTION
**Pull request by ubuntolog**

### CODE REVIEW

File: backend-docker/tests/apiserver_test.go [hunk 1]
```diff
@@ -87,20 +59,65 @@ func callListVolumesHandler(callURL string,  t *testing.T) {
...
 
-		// a string slice to hold the keys
-		k := make([]string, len(c))
-		i := 0
-		// copy c's keys into k
 		for s, _ := range c {
-			k[i] = s
-			i++
+			if isInSlice(s, keyList) !=true {
+				t.Error("The follwoing key was not found in JSON: ", s)
```
Comment (emanueldima) [inline backend-docker/tests/apiserver_test.go]: spelling error: 'following'

Comment (ubuntolog) [inline backend-docker/tests/apiserver_test.go]: I'm glad that was the only comment :)

**Comment by emanueldima** [comment]
rebase?

**Comment by emanueldima** [comment]
this has been rebased on master, see https://github.com/EUDAT-GEF/GEF/commit/6feae97c5a11af7fd4eb6643f56dc401782326b3 and previous commits

**Pull request closed without merging**